ast/src/analyzed/display.rs

```diff
@@ -339,12 +339,63 @@ impl<T: Display> Display for LookupIdentity<T> {
     }
 }
 
+impl<T: Display> Display for PhantomLookupIdentity<T> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        write!(
+            f,
+            "Constr::PhantomLookup(({}, {}), [{}], {});",
+            self.left
+                .selector
+                .as_ref()
+                .map(|s| format!("Some({s})"))
```

Comment:
```suggestion
                .map(|s| format!("Option::Some({s})"))
```
Also below. See my fix in #1953.